processor/integrations/integrations.go

```diff
@@ -23,7 +23,7 @@ func init() {
 
 func loadConfig() {
 	destTransformURL = config.GetEnv("DEST_TRANSFORM_URL", "http://localhost:9090")
-	customDestination = []string{"KINESIS"}
+	customDestination = []string{"KINESIS", "KAFKA"}
```

Comment:
Given that we updated transformers, let's clean this function - GetUserIDFromTransformerResponse

Reply:
Done.